test-app/src/main/java/com/mapbox/navigation/examples/MainActivity.kt

```diff
@@ -0,0 +1,163 @@
+package com.mapbox.navigation.examples
```

Comment:
This file needs to be deleted as it is in `test-app`
@cafesilencio

Reply:
deleted